libnmstate/nm/applier.py

```diff
@@ -325,23 +335,40 @@ def prepare_proxy_ifaces_desired_state(ifaces_desired_state):
         new_ifaces_desired_state.append(port_iface_desired_state)
         # The "visible" slave/interface needs to point to the port profile
         iface_desired_state[MASTER_METADATA] = port_iface_desired_state[
-            Interface.NAME
+            IFACE_NAME_METADATA
         ]
         iface_desired_state[MASTER_TYPE_METADATA] = ovs.PORT_TYPE
     return new_ifaces_desired_state
 
 
 def _create_ovs_port_iface_desired_state(iface_desired_state, port_options):
+    port_name = ovs.PORT_PROFILE_PREFIX + iface_desired_state[Interface.NAME]
     return {
-        "name": ovs.PORT_PROFILE_PREFIX + iface_desired_state[Interface.NAME],
-        "type": ovs.PORT_TYPE,
-        "state": iface_desired_state[Interface.STATE],
+        Interface.NAME: port_name,
+        Interface.TYPE: ovs.PORT_TYPE,
+        Interface.STATE: iface_desired_state[Interface.STATE],
+        OvsB.OPTIONS_SUBTREE: port_options,
         MASTER_METADATA: iface_desired_state[MASTER_METADATA],
         MASTER_TYPE_METADATA: iface_desired_state[MASTER_TYPE_METADATA],
-        "options": port_options,
+        IFACE_NAME_METADATA: _generate_hash_iface_name(port_name),
```

Comment:
This constant name does not really fit IMHO. How about `NM_IFACE_NAME_OVERRIDE`? The current name does not make it clear to me that it is used to override the default name used by NM in its profiles.